editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -2,26 +2,135 @@
 import { useEditorState } from '../../editor/store/store-hook'
 import { MetadataUtils } from '../../../core/model/element-metadata-utils'
 import * as EP from '../../../core/shared/element-path'
 import * as PP from '../../../core/shared/property-path'
 import { jsxAttributeValue } from '../../../core/shared/element-template'
 import { emptyComments } from '../../../core/workers/parser-printer/parser-printer-comments'
 
+const ChromaBlack = chroma('black')
+
+interface TailWindOption {
+  label: string
+  value: string
+  categories?: string[]
+}
+
 const TailWindOptions = TailWindList.classNames.map((className, index) => ({
   label: className,
   value: className,
 }))
 
+const DropdownIndicator = betterReactMemo(
+  'DropdownIndicator',
+  (props: IndicatorProps<TailWindOption, true>) => (
+    <components.DropdownIndicator {...props}>
+      <span> ↓ </span>
+    </components.DropdownIndicator>
+  ),
+)
+
+const ClearIndicator = () => null
+const IndicatorSeparator = () => null
+
+const NoOptionsMessage = betterReactMemo('NoOptionsMessage', (props: any) => (
+  <span {...props}>No results found</span>
+))
+
+const AngledStripe = styled.div({
+  width: 5,
+  height: 22,
+  borderRadius: 0,
+  transform: 'skewX(-11deg)',
+})
+
+const getColorForCategory = (category: string) => {
+  if (category === 'layout') {
+    return '#5FACFF'
+  } else if (category === 'typography') {
+    return '#F7B500'
+  } else if (category === 'interaction') {
+    return '#B620E0'
+  } else return 'pink'
+}
+
+const MultiValueContainer = betterReactMemo(
+  'MultiValueContainer',
+  (props: MultiValueProps<TailWindOption>) => {
+    const { data } = props
+    const stripes: jsx.JSX.Element[] = React.useMemo(() => {
+      const categories = (data?.categories as Array<string>) ?? []
+      if (categories.length > 0) {
+        return categories.map((category, index) => (
+          <AngledStripe
+            key={data?.label ?? index}
+            style={{ backgroundColor: getColorForCategory(category) }}
+          />
+        ))
+      } else {
+        return []
+      }
+    }, [data])
+
+    return (
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          backgroundColor: 'black',
+        }}
+      >
+        <components.MultiValueContainer {...props} />
+        <div
+          style={{
+            display: 'flex',
+            height: 16,
+            paddingRight: 4,
+            paddingLeft: 2,
+          }}
+        >
+          {stripes}
+        </div>
+      </div>
+    )
+  },
+)
+
+const ValueContainer = betterReactMemo(
+  'ValueContainer',
+  (props: ValueContainerProps<TailWindOption, true>) => {
+    return (
+      <div style={{ overflowX: 'scroll', flex: 1 }}>
+        <components.ValueContainer {...props} />
+      </div>
+    )
+  },
+)
+
+const filterOption = (option: Option, rawInput: string) => {
+  return createFilter({ ignoreAccents: false })(option, rawInput)
+}
+
```

Comment:
I think @Rheeseyb was asking for something like this:

```suggestion
const filterIgnoreAccents = createFilter({ ignoreAccents: false })
const filterOption = (option: Option, rawInput: string) => {
  return filterIgnoreAccents(option, rawInput)
}
```